$NetBSD: patch-ab,v 1.3.2.1 2004/07/06 22:32:27 agc Exp $

have a default scsi device which is likely to be correct (at least
on single CD systems).

--- dao/main.cc.orig	2004-06-05 22:57:34.000000000 +0200
+++ dao/main.cc
@@ -115,7 +115,11 @@ static int FIFO_BUFFERS = 32;
 
 #else
 
-static const char *SCSI_DEVICE = "0,0,0";
+#if	defined (__i386__)
+static const char *SCSI_DEVICE = "/dev/rcd0d";
+#else
+static const char *SCSI_DEVICE = "/dev/rcd0c";
+#endif /* __i386__ */
 static int FIFO_BUFFERS = 32;
 
 #endif
